snap/validate_test.go

```diff
@@ -165,6 +165,14 @@ func (s *ValidateSuite) TestValidateAppSocketsEmptyPermsOk(c *C) {
 	c.Check(ValidateApp(app), IsNil)
 }
 
+func (s *ValidateSuite) TestValidateAppSocketsWrongPerms(c *C) {
+	app := createSampleApp()
+	app.Sockets["sock"].SocketMode = 1234
+	err := ValidateApp(app)
+	c.Assert(
```

Comment:
Formatting is weird here. Please put this on one line.

Reply:
done